src/t_stream.c

```diff
@@ -3149,6 +3149,9 @@ void xautoclaimCommand(client *c) {
         server.dirty++;
     }
 
+    /* We need to return the next entry as a cursor for the next XAUTOCLAIM call */
+    raxNext(&ri);
```

Comment:
what if ri is already raxEOF?
maybe add a test where the last possible ID is claimed?

Reply:
@guybe7 - this is already being tested as a part of `XAUTOCLAIM as an iterator`

Reply:
ok i see raxNext early-exits in case the iterator is already EOF